shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -488,9 +491,8 @@ static bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
                                                      transformMatrix)) {
           break;
         }
-        ClipViewSetMaskView(clipView);
-        [(FlutterClippingMaskView*)clipView.maskView clipRect:(*iter)->GetRect()
-                                                       matrix:transformMatrix];
+        clip_rect = transformMatrix.mapRect((*iter)->GetRect());
```

Comment:
This might not be correct, its possible that a skew transform makes a rect non rectangular, we'd need to check that. Also I think once we hit a case where there is a non-clipRect, we'd need to stop using clipsToBounds.

Reply:
There is a mapRect variant that tells you if it was rect-to-rect, I think you have to use `bool mapRect(src,dst)` to get this info. Alternately, SkMatrix has a getter to ask if it preserves rects.

Reply:
Ahh nice, thank you!